### DISCUSSION TITLE
Plugin for DT2 Boss's "Perfect Kill"

### DISCUSSION THREAD

**Discussion by TrippyWash** [runelite/runelite#17345] [Ideas]
I think a great plugin would be a alert or something that showed you if you messed up a perfect kill for all the dt2 bosses. 

Not to be confused with the combat achievements. I'm talking about the 50% increased loot for perfect kills.

**Comment by DevMisery** [comment]
**DT2 Boss Utilities** does this. Displays a info box on the screen with a green checkmark. If you make any mistakes during the fight, it will turn to a red 'X'

https://runelite.net/plugin-hub/show/dt2-mistake-tracker

**Reply by DominickCobb-rs** [reply]
Oh yeah. Forgot I made this to satisfy this request. Thanks for reminding me to tell this person.
